app/helpers/doc_helper.rb

```diff
@@ -1,7 +1,7 @@
 module DocHelper
 
   def man(name)
-    link_to name.gsub(/^git-?/, ''), doc_file_path(:file => name) 
+    link_to name.gsub(/^git-/, ''), doc_file_path(:file => name)
```

Comment:
This effectively reverts d417f6ba159e0d52b2676368482e57e86f604068, which lets us have `revisions` as a link to the `gitrevisions` guide, and so forth. We could solve that with a zero-length lookahead to see if there is any character after `git` (but not substitute it away). Or probably more readably, just turn this function into a series of regex matches.

That said, I actually think some of the guides _do_ want to retain their `git` prefix. E.g., it should be `gitignore` and `gitattributes`. But not `gitrevisions`. So perhaps what we really need is a second optional parameter so callers can override the link text, and to use that for the handful of "guides" that want to drop the `git` prefix.